modules/ROOT/nav.adoc

```diff
@@ -29,10 +29,18 @@ include::{partialsdir}/attributes.adoc[]
 // METRICS
 ** xref:metrics/index.adoc[{metrics-service}]
 *** xref:metrics/provisioning.adoc[Provisioning]
-*** xref:metrics/binding.adoc[Binding]
-*** xref:metrics/configuring-dev-env-metrics.adoc[Configuring a Dev environment]
-*** xref:metrics/analysing-app-usage.adoc[Analysing App Usage]
-*** xref:monitoring-keycloak.adoc[Monitoring {keycloak-service}]
+*** Client metrics
+**** xref:metrics/binding.adoc[Binding]
+**** xref:metrics/configuring-dev-env-metrics.adoc[Configuring a Dev environment]
+**** xref:metrics/analysing-app-usage.adoc[Analysing App Usage]
+*** Service metrics
+**** xref:metrics/monitoring-keycloak.adoc[Monitoring {keycloak-service}]
+*** More...
+**** xref:metrics/giving-an-openshift-user-access-to-metrics.adoc[Sharing access]
+**** xref:metrics/making-mobile-services-discoverable-by-metrics.adoc[Auto-discovery]
```

Comment:
The sections `Auto-discovery` and `Manual discovery` are not that useful for an end-user unless they are writing their own services. So we could either leave and link them from the custom runtime connector docs or remove them. What do you think @finp @jstaffor  ?

Reply:
Similarly, we have "Sharing Access" document which is way too technical.

Reply:
I was reading the "Using Dashboards" doc and it is mostly about how to develop a dashboard. We should also remove that.